tests/integration/diff_test.go

```diff
@@ -154,6 +228,24 @@ func Test_Diff_Masked_OlderThan3x(t *testing.T) {
 			assert.Equal(t, expectedOutputMasked, out)
 		})
 	}
+
+	for _, tc := range tests {
```

Comment:
Testing this locally (`make setup-kong test-integration`) shows a failure for these tests. The reason is that the new output generated by the `--json-output` functionality is not properly captured by the test utils.

I'm okay with removing this test for now and add it back later on, but I also tested this manually and I noticed that env variables are not masked when the new flag is used.

If you want to work on this later on, please feel free to open a new GitHub issue, but in that case we need to call out this behaviour in the [flag's description](https://github.com/Kong/deck/blob/26372483a00e6cc52c336573e1010d67c46a0745/cmd/sync.go#L66).

WDYT?

Reply:
Will resolve this as the masking was fixed. :)